services/api/src/domain/messages.ts

```diff
@@ -1,6 +1,7 @@
 import { GameWebSocket } from './GameRoom';
 
 export enum MessageType {
+  PlayerSuccessfullyJoined = 'PlayerSuccessfullyJoined',
```

Comment:
What is the difference between `PlayerSuccessfullyJoined` and  `PlayerConnected`?

Reply:
PlayerConnected is broadcasted to everyone, when new player joins, updating player list, ect..
PlayerSuccessfullyJoined is send only to the player who joined in order to save his id in frontend.